### PR TITLE
adding rasterPointCount function and help file

### PR DESCRIPTION
**Pull request by fshearer**

### CODE REVIEW

File: man/rasterPointCount.Rd [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+rasterPointCount(rasterbrick, coords)
+}
+\arguments{
+  \item{rasterbrick}{
+The raster from which to calculate the number of points within each pixel.
+}
+  \item{coords}{
+A two-column matrix of coordinates in the order x then y.
+}
+
```
Comment (goldingn) [inline man/rasterPointCount.Rd]: you're missing a curly bracket `}` to close `\arguments` here

File: man/rasterPointCount.Rd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+\value{
+A two-column dataframe with each row containing the pixel ID number and the count of points for that pixel. 
+\seealso{
```
Comment (goldingn) [inline man/rasterPointCount.Rd]: ...and you're missing a curly bracket `}` here to close `\value` (i.e. before `\seealso`)

**Comment by goldingn** [comment]
There are a couple of errors in the documentation which prevent this from building properly (see comments above).

The check also fails on the example for this function, because it is unexported. You need to edit NAMESPACE to explicitly export the function.

If you make these changes locally and then push to your fork it should update the PR. then you can leave me a comment here to let me know it's ready to go!

**Comment by fshearer** [comment]
Fix it and passes checks

**Comment by goldingn** [comment]
:shipit:

**Comment by goldingn** [comment]
http://www.quora.com/GitHub/What-is-the-significance-of-the-Ship-It-squirrel

**Pull request merged**